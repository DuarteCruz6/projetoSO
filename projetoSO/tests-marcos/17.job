WRITE [(y5,6)]
READ [y5]
WRITE [(x6,67)(g6,10)(u3,34)]
READ [u3]
READ [x6]
WRITE [(c3,6)(d1,85)(s8,14)(j5,71)(v9,11)]
DELETE [v9,g6]
READ [y5]
READ [c3]
WRITE [(k2,13)(g3,63)(o1,86)]
WRITE [(c7,60)]
WRITE [(n6,92)(k8,15)(b8,71)]
DELETE [g3,j5,c7]
WRITE [(i9,57)(j4,52)(q2,51)(n5,41)(q3,44)]
SHOW
WRITE [(r8,49)(g8,95)(l6,17)]
WRITE [(j7,1)(i8,42)(z6,48)(p1,14)]
READ [i9]
WRITE [(s9,65)(l9,25)(z5,67)(j8,24)(h1,33)]
READ [i9]
WRITE [(m8,17)(y9,25)]
WRITE [(v2,28)(g4,9)(w4,70)(g2,29)(d9,96)]
WRITE [(n1,77)(k5,70)(v7,31)]
WRITE [(f5,66)(r0,23)(s7,44)(i6,8)(c2,1)]
WAIT 2000
WRITE [(f2,49)(k3,94)]
READ [k3]
DELETE [r0,x6]
READ [k3]
WRITE [(d2,91)(b4,80)(j0,1)]
WRITE [(w3,14)(m1,30)(l4,46)]
WRITE [(m3,44)(m5,68)]
WRITE [(h5,4)(t2,85)]
READ [i8]
WRITE [(g5,81)]
DELETE [w4]
WRITE [(k0,50)(x4,87)(z2,88)(d3,100)]
DELETE [s9,n1,c3]
WRITE [(k7,9)(n3,4)]
WRITE [(w0,44)(e2,1)(s2,64)(g7,47)(x5,90)]
DELETE [s7,y9,o1,k2,z6]
DELETE [f5,f2,c2]
DELETE [d1,q3,n6]
WRITE [(a2,66)]
DELETE [m5,r8]
WRITE [(o3,5)(r7,21)(x0,62)(v0,44)(v8,65)]
READ [k0]
WRITE [(r1,49)(a9,21)(h6,25)]
WRITE [(w6,93)(u6,52)(u9,41)(q7,13)]
WRITE [(c2,33)]
READ [i9]
WRITE [(j2,24)(e0,81)(d0,90)]
WRITE [(f9,16)]
WRITE [(x8,43)(v4,92)(j5,93)(b9,31)(o0,82)]
WRITE [(f8,70)(e9,14)]
DELETE [n3,o0,l4,x5]
READ [h1]
DELETE [g2,z2,a2,v4]
READ [q2]
READ [a9]
DELETE [j5,b9]
BACKUP
DELETE [u6,l9,i8,r7,i9]
WRITE [(s6,10)(u1,83)(m0,23)(j1,95)(h8,80)]
WRITE [(z2,65)(x3,56)(t9,42)(r2,84)(a7,85)]
DELETE [k8,j0,e2]
DELETE [d9,g8,v7]
DELETE [l6]
WRITE [(i1,32)(w7,52)(q0,58)(j3,19)(q4,56)]
WRITE [(x5,3)(h7,30)]
READ [j2]
WAIT 2000
WRITE [(r0,24)(o6,41)(g0,96)]
WRITE [(z6,37)(w8,8)(o9,45)]
WRITE [(v7,4)(l1,27)]
DELETE [a9,m8,r2]
DELETE [j3,t2,e9,e0]
WRITE [(s4,33)(c4,91)]
WRITE [(c5,7)(o0,86)(a1,3)(i3,58)(t8,21)]
WRITE [(c8,8)(c8,16)(m8,86)(e6,15)]
BACKUP
READ [x4]
READ [w3]
READ [g4]
WRITE [(i0,37)(y0,11)(p6,87)(f6,57)(o7,89)]
BACKUP
DELETE [q7,h5]
WRITE [(d6,19)(a3,64)(d6,24)(z9,80)]
WAIT 1000
WRITE [(r4,87)(f1,87)]
WRITE [(u7,36)(f5,59)(j0,75)(x2,35)(h4,9)]
WRITE [(b1,96)]
READ [v7]
READ [j1]
WRITE [(p4,55)]
WAIT 2000
READ [z5]
