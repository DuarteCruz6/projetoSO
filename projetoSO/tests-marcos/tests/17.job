WRITE [(q0,21)(h6,3)(g5,66)]
WRITE [(w7,38)(h4,81)(n6,100)(r4,88)]
READ [w7]
WAIT 3000
WRITE [(h5,54)]
READ [n6]
SHOW
DELETE [w7,n6]
WRITE [(m6,91)(p1,53)(c3,32)(j2,94)(f9,50)]
WAIT 1000
WRITE [(h0,79)]
WAIT 1000
WRITE [(f8,33)(z7,4)]
WRITE [(d0,94)(l4,54)(i3,42)(q3,58)]
WRITE [(t5,98)]
WRITE [(i5,87)]
WRITE [(j6,15)(b6,45)(n3,87)]
WRITE [(v2,87)(s2,1)(b3,29)]
READ [b6]
WRITE [(m0,20)(g6,63)(h9,95)(t8,41)(e9,82)]
DELETE [t5]
WRITE [(m8,94)]
WRITE [(g0,2)]
WAIT 2000
DELETE [s2]
WRITE [(o8,42)(j1,22)(t1,42)(z8,31)]
WRITE [(t6,74)(k2,61)]
WRITE [(i6,30)(t0,44)(i6,74)(m2,54)]
WRITE [(p7,22)(j8,48)(t4,93)(q7,59)(j9,65)]
WRITE [(n1,28)(f5,58)(f0,47)(e6,57)(d1,66)]
WAIT 1000
WAIT 2000
BACKUP
WRITE [(z5,65)]
WRITE [(i9,30)(b5,10)(k1,100)(d3,10)(f7,98)]
READ [e6]
WAIT 1000
SHOW
READ [t6]
WRITE [(y4,74)(c7,19)(b1,84)(o2,29)(o7,5)]
WRITE [(a7,74)(r8,33)]
WRITE [(z1,87)(p3,84)(u3,0)(v7,74)]
DELETE [b5,l4,j9,z8,d1]
DELETE [q0,m2,g5]
WRITE [(e1,69)(g3,19)(b2,86)]
WRITE [(o4,41)(q5,50)(l1,25)(l7,39)(z3,27)]
DELETE [b3]
BACKUP
DELETE [e6]
WRITE [(q0,18)(l3,50)]
READ [z7]
WAIT 2000
WRITE [(r6,98)(e4,57)(q9,28)]
WRITE [(o6,30)]
READ [p3]
READ [b2]
READ [b2]
WRITE [(n2,49)]
WAIT 2000
WRITE [(s0,46)(g2,40)(o0,33)(v5,8)]
WRITE [(e8,32)(x2,57)(v4,16)(a8,7)(j3,37)]
READ [e9]
READ [n2]
SHOW
SHOW
WRITE [(n9,63)(y9,12)(c1,63)(l9,4)(k6,97)]
READ [z5]
WRITE [(u7,100)(g4,66)(p6,83)(h1,19)(z9,91)]
SHOW
WRITE [(m3,56)(k9,34)(s9,86)(o9,72)(l5,25)]
WRITE [(s8,21)(a4,83)(i4,67)]
DELETE [a4,n9,j6]
READ [i6]
READ [y4]
WRITE [(u1,75)(u9,59)]
SHOW
WRITE [(s5,89)(d6,100)(v0,49)(j4,1)]
WRITE [(f6,35)(s4,27)(r0,12)(n7,89)(e5,83)]
DELETE [l1,l3,o2]
WRITE [(x0,66)(n5,57)(e0,77)]
WRITE [(d4,43)(u2,7)(l6,10)(m7,66)(d4,94)]
WRITE [(w8,60)(g8,79)]
WAIT 3000
WRITE [(h2,93)]
READ [h9]
BACKUP
DELETE [t1,t0]
DELETE [l7]
READ [f8]
WRITE [(q2,7)(t7,1)(t7,97)(t0,80)(p5,18)]
WRITE [(d8,54)(s3,82)(w1,17)(j0,63)]
WRITE [(k5,98)(u6,30)(i7,88)(j6,28)]
DELETE [n3,v5,m6]
WRITE [(c2,2)(y8,52)(r5,22)(w2,49)(n6,81)]
READ [j0]
BACKUP
READ [p5]
SHOW
WRITE [(n8,61)(n0,58)(d9,18)]
WRITE [(l7,0)(m6,72)]
