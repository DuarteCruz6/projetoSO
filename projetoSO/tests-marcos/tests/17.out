[(w7,38)]
[(n6,100)]
(g5, 66)
(h5, 54)
(h4, 81)
(h6, 3)
(n6, 100)
(q0, 21)
(r4, 88)
(w7, 38)
[(b6,45)]
[(e6,57)]
(b5, 10)
(b3, 29)
(b6, 45)
(c3, 32)
(d3, 10)
(d1, 66)
(d0, 94)
(e6, 57)
(e9, 82)
(f7, 98)
(f0, 47)
(f5, 58)
(f8, 33)
(f9, 50)
(g0, 2)
(g6, 63)
(g5, 66)
(h9, 95)
(h0, 79)
(h5, 54)
(h4, 81)
(h6, 3)
(i9, 30)
(i6, 74)
(i5, 87)
(i3, 42)
(j9, 65)
(j8, 48)
(j1, 22)
(j6, 15)
(j2, 94)
(k1, 100)
(k2, 61)
(l4, 54)
(m2, 54)
(m8, 94)
(m0, 20)
(m6, 91)
(n1, 28)
(n3, 87)
(o8, 42)
(p7, 22)
(p1, 53)
(q7, 59)
(q3, 58)
(q0, 21)
(r4, 88)
(t4, 93)
(t0, 44)
(t6, 74)
(t1, 42)
(t8, 41)
(v2, 87)
(z5, 65)
(z8, 31)
(z7, 4)
[(t6,74)]
[(z7,4)]
[(p3,84)]
[(b2,86)]
[(b2,86)]
[(e9,82)]
[(n2,49)]
(a8, 7)
(a7, 74)
(b2, 86)
(b1, 84)
(b6, 45)
(c7, 19)
(c3, 32)
(d3, 10)
(d0, 94)
(e8, 32)
(e4, 57)
(e1, 69)
(e9, 82)
(f7, 98)
(f0, 47)
(f5, 58)
(f8, 33)
(f9, 50)
(g2, 40)
(g3, 19)
(g0, 2)
(g6, 63)
(h9, 95)
(h0, 79)
(h5, 54)
(h4, 81)
(h6, 3)
(i9, 30)
(i6, 74)
(i5, 87)
(i3, 42)
(j3, 37)
(j8, 48)
(j1, 22)
(j6, 15)
(j2, 94)
(k1, 100)
(k2, 61)
(l3, 50)
(l7, 39)
(l1, 25)
(m8, 94)
(m0, 20)
(m6, 91)
(n2, 49)
(n1, 28)
(n3, 87)
(o0, 33)
(o6, 30)
(o4, 41)
(o7, 5)
(o2, 29)
(o8, 42)
(p3, 84)
(p7, 22)
(p1, 53)
(q9, 28)
(q0, 18)
(q5, 50)
(q7, 59)
(q3, 58)
(r6, 98)
(r8, 33)
(r4, 88)
(s0, 46)
(t4, 93)
(t0, 44)
(t6, 74)
(t1, 42)
(t8, 41)
(u3, 0)
(v4, 16)
(v5, 8)
(v7, 74)
(v2, 87)
(x2, 57)
(y4, 74)
(z3, 27)
(z1, 87)
(z5, 65)
(z7, 4)
(a8, 7)
(a7, 74)
(b2, 86)
(b1, 84)
(b6, 45)
(c7, 19)
(c3, 32)
(d3, 10)
(d0, 94)
(e8, 32)
(e4, 57)
(e1, 69)
(e9, 82)
(f7, 98)
(f0, 47)
(f5, 58)
(f8, 33)
(f9, 50)
(g2, 40)
(g3, 19)
(g0, 2)
(g6, 63)
(h9, 95)
(h0, 79)
(h5, 54)
(h4, 81)
(h6, 3)
(i9, 30)
(i6, 74)
(i5, 87)
(i3, 42)
(j3, 37)
(j8, 48)
(j1, 22)
(j6, 15)
(j2, 94)
(k1, 100)
(k2, 61)
(l3, 50)
(l7, 39)
(l1, 25)
(m8, 94)
(m0, 20)
(m6, 91)
(n2, 49)
(n1, 28)
(n3, 87)
(o0, 33)
(o6, 30)
(o4, 41)
(o7, 5)
(o2, 29)
(o8, 42)
(p3, 84)
(p7, 22)
(p1, 53)
(q9, 28)
(q0, 18)
(q5, 50)
(q7, 59)
(q3, 58)
(r6, 98)
(r8, 33)
(r4, 88)
(s0, 46)
(t4, 93)
(t0, 44)
(t6, 74)
(t1, 42)
(t8, 41)
(u3, 0)
(v4, 16)
(v5, 8)
(v7, 74)
(v2, 87)
(x2, 57)
(y4, 74)
(z3, 27)
(z1, 87)
(z5, 65)
(z7, 4)
[(z5,65)]
(a8, 7)
(a7, 74)
(b2, 86)
(b1, 84)
(b6, 45)
(c1, 63)
(c7, 19)
(c3, 32)
(d3, 10)
(d0, 94)
(e8, 32)
(e4, 57)
(e1, 69)
(e9, 82)
(f7, 98)
(f0, 47)
(f5, 58)
(f8, 33)
(f9, 50)
(g4, 66)
(g2, 40)
(g3, 19)
(g0, 2)
(g6, 63)
(h1, 19)
(h9, 95)
(h0, 79)
(h5, 54)
(h4, 81)
(h6, 3)
(i9, 30)
(i6, 74)
(i5, 87)
(i3, 42)
(j3, 37)
(j8, 48)
(j1, 22)
(j6, 15)
(j2, 94)
(k6, 97)
(k1, 100)
(k2, 61)
(l9, 4)
(l3, 50)
(l7, 39)
(l1, 25)
(m8, 94)
(m0, 20)
(m6, 91)
(n9, 63)
(n2, 49)
(n1, 28)
(n3, 87)
(o0, 33)
(o6, 30)
(o4, 41)
(o7, 5)
(o2, 29)
(o8, 42)
(p6, 83)
(p3, 84)
(p7, 22)
(p1, 53)
(q9, 28)
(q0, 18)
(q5, 50)
(q7, 59)
(q3, 58)
(r6, 98)
(r8, 33)
(r4, 88)
(s0, 46)
(t4, 93)
(t0, 44)
(t6, 74)
(t1, 42)
(t8, 41)
(u7, 100)
(u3, 0)
(v4, 16)
(v5, 8)
(v7, 74)
(v2, 87)
(x2, 57)
(y9, 12)
(y4, 74)
(z9, 91)
(z3, 27)
(z1, 87)
(z5, 65)
(z7, 4)
[(i6,74)]
[(y4,74)]
(a8, 7)
(a7, 74)
(b2, 86)
(b1, 84)
(b6, 45)
(c1, 63)
(c7, 19)
(c3, 32)
(d3, 10)
(d0, 94)
(e8, 32)
(e4, 57)
(e1, 69)
(e9, 82)
(f7, 98)
(f0, 47)
(f5, 58)
(f8, 33)
(f9, 50)
(g4, 66)
(g2, 40)
(g3, 19)
(g0, 2)
(g6, 63)
(h1, 19)
(h9, 95)
(h0, 79)
(h5, 54)
(h4, 81)
(h6, 3)
(i4, 67)
(i9, 30)
(i6, 74)
(i5, 87)
(i3, 42)
(j3, 37)
(j8, 48)
(j1, 22)
(j2, 94)
(k9, 34)
(k6, 97)
(k1, 100)
(k2, 61)
(l5, 25)
(l9, 4)
(l3, 50)
(l7, 39)
(l1, 25)
(m3, 56)
(m8, 94)
(m0, 20)
(m6, 91)
(n2, 49)
(n1, 28)
(n3, 87)
(o9, 72)
(o0, 33)
(o6, 30)
(o4, 41)
(o7, 5)
(o2, 29)
(o8, 42)
(p6, 83)
(p3, 84)
(p7, 22)
(p1, 53)
(q9, 28)
(q0, 18)
(q5, 50)
(q7, 59)
(q3, 58)
(r6, 98)
(r8, 33)
(r4, 88)
(s8, 21)
(s9, 86)
(s0, 46)
(t4, 93)
(t0, 44)
(t6, 74)
(t1, 42)
(t8, 41)
(u9, 59)
(u1, 75)
(u7, 100)
(u3, 0)
(v4, 16)
(v5, 8)
(v7, 74)
(v2, 87)
(x2, 57)
(y9, 12)
(y4, 74)
(z9, 91)
(z3, 27)
(z1, 87)
(z5, 65)
(z7, 4)
[(h9,95)]
[(f8,33)]
[(j0,63)]
[(p5,18)]
(a8, 7)
(a7, 74)
(b2, 86)
(b1, 84)
(b6, 45)
(c2, 2)
(c1, 63)
(c7, 19)
(c3, 32)
(d8, 54)
(d4, 94)
(d6, 100)
(d3, 10)
(d0, 94)
(e0, 77)
(e5, 83)
(e8, 32)
(e4, 57)
(e1, 69)
(e9, 82)
(f6, 35)
(f7, 98)
(f0, 47)
(f5, 58)
(f8, 33)
(f9, 50)
(g8, 79)
(g4, 66)
(g2, 40)
(g3, 19)
(g0, 2)
(g6, 63)
(h2, 93)
(h1, 19)
(h9, 95)
(h0, 79)
(h5, 54)
(h4, 81)
(h6, 3)
(i7, 88)
(i4, 67)
(i9, 30)
(i6, 74)
(i5, 87)
(i3, 42)
(j6, 28)
(j0, 63)
(j4, 1)
(j3, 37)
(j8, 48)
(j1, 22)
(j2, 94)
(k5, 98)
(k9, 34)
(k6, 97)
(k1, 100)
(k2, 61)
(l6, 10)
(l5, 25)
(l9, 4)
(m7, 66)
(m3, 56)
(m8, 94)
(m0, 20)
(n6, 81)
(n5, 57)
(n7, 89)
(n2, 49)
(n1, 28)
(o9, 72)
(o0, 33)
(o6, 30)
(o4, 41)
(o7, 5)
(o8, 42)
(p5, 18)
(p6, 83)
(p3, 84)
(p7, 22)
(p1, 53)
(q2, 7)
(q9, 28)
(q0, 18)
(q5, 50)
(q7, 59)
(q3, 58)
(r5, 22)
(r0, 12)
(r6, 98)
(r8, 33)
(r4, 88)
(s3, 82)
(s4, 27)
(s5, 89)
(s8, 21)
(s9, 86)
(s0, 46)
(t0, 80)
(t7, 97)
(t4, 93)
(t6, 74)
(t8, 41)
(u6, 30)
(u2, 7)
(u9, 59)
(u1, 75)
(u7, 100)
(u3, 0)
(v0, 49)
(v4, 16)
(v7, 74)
(v2, 87)
(w2, 49)
(w1, 17)
(w8, 60)
(x0, 66)
(x2, 57)
(y8, 52)
(y9, 12)
(y4, 74)
(z9, 91)
(z3, 27)
(z1, 87)
(z5, 65)
(z7, 4)
